tools/validators/ontology_validator/yamlformat/validator/presubmit_validate_types_lib.py

```diff
@@ -560,8 +560,7 @@ def RunPresubmit(unmodified, modified_base, modified_client):
 
   """
 
-  findings, _ = _ValidateConfigInner(unmodified, modified_base, modified_client)
-
+  findings, _ = _ValidateConfigInner(unmodified, modified_base, modified_client, False)
```

Comment:
Out of curiosity, is it better style to do `interactive=False` for the last argument? But on line `639` it just inputs the boolean so maybe this is the convention of the codebase.